src/main/java/spoon/reflect/declaration/CtModule.java

```diff
@@ -70,13 +70,17 @@
 
 	@PropertySetter(role = MODULE_DIRECTIVE)
 	<T extends CtModule> T addModuleDirectiveAt(int position, CtModuleDirective moduleDirective);
-
+	/**
+	 * @return an unmodifiable view of module directives
```

Comment:
The whole class has zero doc? Maybe someone finds time to document all, if he has knowledge about modules.